protos/capture_data.proto

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto3";
+
+package protos;
+
+message FunctionStats {
+  uint64 count = 1;
+  uint64 total_time_ns = 2;
+  uint64 average_time_ns = 3;
+  uint64 min_ns = 4;
+  uint64 max_ns = 5;
+}
```

Comment:
maybe move these to OrbitGl? they are not used on the service side are they?

Reply:
These messages aren't used on the service side. Does `protos` only for service? I thought that it's for all the protos, and because of it placed `capture_data` here. But talking about moving it to another module: why to OrbitGl and not to OrbitCore? We use CallstackEvent and FunctionInfo in OrbitCore, so probably we should move to OrbitCore or change the logic.